packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+- BugFix: Fix in employing `--context-lines` option for all diff/compare commands. Fixed broken `--seqnum` option implementation. Corrected and added unit, system and integration tests [#1529](https://github.com/zowe/zowe-cli/issues/1529)
```

Comment:
We mention only user-facing changes in the changelog, so no need to mention "Corrected and added unit, system and integration tests" here 🙂